gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/impl/AlertServiceImpl.java

```diff
@@ -596,6 +607,10 @@ private void setDefaultEmailNotifier(Notification notification) {
             configuration.setSslTrustAll(sslTrustAll);
         }
 
+        Set<String> authMethods = ofNullable(this.authMethods).map(s -> s.split(" +")).map(Set::of).orElse(null);
```

Comment:
Usually we separate items in a list with a comma and not with a 'blank'.

And by doing this, you could simply have `this.authMethods` as a String array like this (see https://www.baeldung.com/spring-inject-arrays-lists):

```
@Value("${notifiers.email.authMethods:#{null}}") String[] authMethods,
```

And as the `configuration.authMethods` is meant to be serialized in JSON, maybe a string array would be enough and you would not need to convert to a Set.

Reply:
Thanks for the review !

> And by doing this, you could simply have this.authMethods as a String array like this (see https://www.baeldung.com/spring-inject-arrays-lists):
> @Value("${notifiers.email.authMethods:#{null}}") String[] authMethods,

Agreed with this since this is cleaner

> And as the configuration.authMethods is meant to be serialized in JSON, maybe a string array would be enough and you would not need to convert to a Set.

I still care about uniqueness of values and `configuration.authMethod` is already defined this way in the email notifier 1.5.0 with a Set so I'll keep it this way for consistency. 
Nevertheless it's still better to do this from an array